<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>188</width>
    <height>251</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="dungeon2DButton">
        <property name="text">
         <string>2D dungeon</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="dungeon3DButton">
        <property name="text">
         <string>3D dungeon</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="yDimensionLabel">
        <property name="text">
         <string>Total Dungeon Length</string>
        </property>
        <property name="buddy">
         <cstring>yDimensionSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="xDimensionLabel">
        <property name="text">
         <string>Total Dungeon Width</string>
        </property>
        <property name="buddy">
         <cstring>xDimensionSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="zDimensionSpinBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="zDimensionLabel">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Number of Levels</string>
        </property>
        <property name="buddy">
         <cstring>zDimensionSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="yDimensionSpinBox">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="xDimensionSpinBox">
        <property name="toolTip">
         <string/>
        </property>
        <property name="toolTipDuration">
         <number>-1</number>
        </property>
        <property name="statusTip">
         <string/>
        </property>
        <property name="whatsThis">
         <string/>
        </property>
        <property name="accessibleName">
         <string/>
        </property>
        <property name="accessibleDescription">
         <string/>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhDigitsOnly</set>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="xRoomDimensionSpinBox">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="xRoomDimensionLabel">
        <property name="text">
         <string>Maximum Room Width</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="yRoomDimensionLabel">
        <property name="text">
         <string>Maximum Room Length</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="yRoomDimensionSpinBox">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="standardButtons">
       <set>QDialogButtonBox::Ok|QDialogButtonBox::Reset</set>
      </property>
      <property name="centerButtons">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>188</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Dungeon"/>
    <addaction name="actionLoad_Dungeon"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew_Dungeon">
   <property name="text">
    <string>New Dungeon</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionLoad_Dungeon">
   <property name="text">
    <string>Load Dungeon</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>dungeon2DButton</tabstop>
  <tabstop>dungeon3DButton</tabstop>
  <tabstop>xDimensionSpinBox</tabstop>
  <tabstop>yDimensionSpinBox</tabstop>
  <tabstop>zDimensionSpinBox</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
